### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by rocavalc**
## PR Checklist
* [ ] Closes Issue #xxx
* [ ] Code of Conduct signed. If not, go to [Code of Conduct](https://github.com/microsoft/industry/blob/main/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Review by rocavalc — COMMENTED** [review]
I reviewed it and it appears to be right. Need someone else to check it out

**Comment by amanjeetsingh** [comment]
Reviewing now. Thanks!

File: retail/solutions/churn/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+- Create custom reports and views based on real-time customer behavior data, leverage built-in web and mobile analytics to predict customer needs. 
+
+
+Microsoft Cloud for Retail Unified customer profile includes in the U.S. an **AI-based churn predictive model**, designed for omnichannel retail and built atop Customer Insights. Create a retail channel churn predictive model to fit your business needs, and gain cross-channel insights into the chance of retail customer churn. Run your company data through this model, training it to improve its predictions and identify the factors that contribute to churn, at the customer level.
+
+## Prerequisites
+
+- Retail components, available within Microsoft Cloud for Retail in Microsoft Cloud Solution Center. For more information about deploying these components, go to [Deploy Unified customer profile in Microsoft Cloud for Retail](https://docs.microsoft.com/en-us/dynamics365/industry/retail/retail-deploy-unified-customer-profile).
+
+- At least Contributor permissions in Microsoft Dynamics 365 Customer Insights. For more information, go to User permissions.
```
Comment (amanjeetsingh) [inline retail/solutions/churn/README.md:34]: Probably better to just leave it as "Contributor role" and remove "at least".

File: retail/solutions/churn/README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+- Gain more nuanced insights by combining digital analytics with customer profiles to create richer segments, and leverage churn models to **understand churn risk** at a glance. 
+
+- Observe customer progress through each defined step of the journey, quickly identifying obstacles and opportunities. 
+
+- Create custom reports and views based on real-time customer behavior data, leverage built-in web and mobile analytics to predict customer needs. 
+
+
+Microsoft Cloud for Retail Unified customer profile includes in the U.S. an **AI-based churn predictive model**, designed for omnichannel retail and built atop Customer Insights. Create a retail channel churn predictive model to fit your business needs, and gain cross-channel insights into the chance of retail customer churn. Run your company data through this model, training it to improve its predictions and identify the factors that contribute to churn, at the customer level.
+
+## Prerequisites
```
Comment (amanjeetsingh) [inline retail/solutions/churn/README.md:30]: I think we should explicitly call out dependency on **D365 Customer Insights** and **MS Cloud for Retail Add-On**. Thoughts?

**Review by amanjeetsingh — COMMENTED** [review]
@rocavalc - can we please change structure of this page to align with the structure we've used for  other solution guidance docs? See FSI, Healthcare or other retail solutions (under /retail/solutions) directory? Thanks

File: retail/solutions/README.md [hunk 3]
```diff
@@ -1,14 +1,16 @@
 # Retail Solutions
```
Comment (amanjeetsingh) [inline retail/solutions/README.md:1]: @rocavalc , as discussed during our chat last night, let's change this to Unified Customer Profile (UCP).
UCP will be the parent topic and within this, we can have a separate section to cover "Retail channel churn prediction". This structure will be in line with the actual composition of UCP. Thanks.

Comment (amanjeetsingh) [inline retail/solutions/README.md:1]: @rocavalc , as discussed during our chat last night, let's change this to Unified Customer Profile (UCP).
UCP will be the parent topic and within this, we can have a separate section to cover "Retail channel churn prediction". This structure will be in line with the actual composition of UCP. Thanks.

**Pull request closed without merging**